airbyte-integrations/bases/source-acceptance-test/.coveragerc

```diff
@@ -0,0 +1,7 @@
+[report]
+# show lines missing coverage
+show_missing = true
+# coverage 63% measured on a54c0efc8d70df427952901a59730e657f04ff0a (2022-08-09)
+fail_under = 63
```

Comment:
I like this a lot. Reminds me of browser performance tests where the build asserts that page render times never go down. Added a suggestion
```suggestion
# This number should probably never be adjusted down, only up i.e: we should only ever increase our test coverage
fail_under = 63
```